src/pages/AddData.tsx

```diff
@@ -31,6 +38,7 @@ interface BulkUploaderProps {
 	onBulkUpload: () => void;
 }
 const BulkUploader = ({ onBulkUpload }: BulkUploaderProps) => {
+	const [sources] = useSources(SOURCES_DEFAULT_PAGE, SOURCES_DEFAULT_COUNT);
```

Comment:
The problem that @hminsky2002 called out with this is that if the system source does not show up in this call, we error down below.

This is exasperated by two factors: using a synthetic key so that we need to do this at all, and that the `/sources` endpoint returns the most recent sources (sorted by id descending) rather than the earliest sources, as the system source should always be the first source.

The change that introduced the system source, https://github.com/PhilanthropyDataCommons/service/pull/1116, created the table and inserted the system source row in the same migration, and the id is `GENERATED ALWAYS AS IDENTITY`, so the system source ID should always be `1` and cannot be `UPDATE`d. Maybe it's worth hardcoding that here? cc @slifty 

```psql
pdc=*> update sources set id = 2;
ERROR:  column "id" can only be updated to DEFAULT
DETAIL:  Column "id" is an identity column defined as GENERATED ALWAYS.
```

Reply:
@jasonaowen that's fine; in the long run IMHO it would be appropriate to provide a documented API mechanism to load the system source rather than relying on magic values.

Reply:
@slifty that could be good, although it's not clear to me how often it'd be used or if it'd be used by other partners or only for the type of admin work that our team is still forecast to do. If it's just us, a magic number is fine, and `1` is the second least-magical number! If we could encode that in the SDK somehow, or at least document it, that would be good.

For now, @hminsky2002, I suggest a new `useSystemSource` hook that fetches `/sources/1`, so that you don't have to worry about search or pagination or anything like that.